frontend/packages/gitops-plugin/src/components/utils/gitops-utils.ts

```diff
@@ -62,11 +62,15 @@ export const fetchAllAppGroups = async (baseURL: string, manifestURLs: string[],
   return [allAppGroups, emptyMsg];
 };
 
-export const getEnvData = async (envURI: string, env: string, appURI: string) => {
+export const getEnvData = async (v2EnvURI: string, envURI: string, env: string, appURI: string) => {
   let data;
```

Comment:
you don't have a double fallback for this. Is `undefined` the desired `getEnvData`?